codemeta.json

```diff
@@ -0,0 +1,83 @@
+{
+    "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+    "@type": "SoftwareSourceCode",
+    "license": "https://spdx.org/licenses/GPL-3.0",
+    "codeRepository": "https://github.com/javierrico/gLike",
+    "dateCreated": "2018-11-13",
+    "datePublished": "2020-07-30",
+    "dateModified": "2021-03-12",
+    "downloadUrl": "https://github.com/javierrico/gLike/releases/tag/v00.09.03",
```

Comment:
Does this mean that we need to update this file for every release?

Reply:
yes, you should update each time with the date and version of the release manually.
This correspond to the "Current version of the software" in the [codemeta generator](https://codemeta.github.io/codemeta-generator/).